src/gui/plugins/component_inspector/AirPressure.cc

```diff
@@ -30,7 +30,7 @@ using namespace gazebo;
 /////////////////////////////////////////////////
 AirPressure::AirPressure(ComponentInspector *_inspector)
 {
-  _inspector->Context()->setContextProperty("AirPressure", this);
+  _inspector->Context()->setContextProperty("AirPressureImpl", this);
```

Comment:
Does the name collision come from the fact that we are adding property (typically object instances) using `CamelCase` which is what we normally use for class names?

Reply:
My guess is that it's a collision between this context property, and the `AirPressure.qml` item itself